### ISSUE TITLE
Uncaught `null` in Chrome Inspector

### ISSUE THREAD

**Issue by radeusgd** [enso-org/enso#6270]
I was able to get an error in the Chrome Inspector REPL when debugging an Enso script.

The repro is following:

Run the following script (I called it `too-many-warnings.enso`) with the `--inspect` option:
```
from Standard.Base import all
from Standard.Table import Table, Value_Type
from Standard.Database import Database, SQLite, In_Memory

main =
    c = Database.connect (SQLite In_Memory)
    t0 = Table.new [["X", ["a", "bc", "def"]]]
    t1 = c.upload_table "Tabela" t0
    t2 = t1.cast "X" (Value_Type.Char size=1)
    t2.print
    warnings = Warning.get_all t2 . map .value
    IO.println warnings.length
    IO.println warnings.first
    IO.println warnings.distinct.length
```

Set a breakpoint at line `10`: `t2.print` and run up to this line.

When in this line, observe the following interaction in the REPL:
```
> t1.at "X" . value_type
  <- Value_Type (Char Nothing True)
> t2.at "X" . value_type
  <- Uncaught null
    at too-many-warnings.main (Unknown)
```

<img width="951" alt="image" src="https://user-images.githubusercontent.com/1436948/231778421-20236f65-acc1-444f-8ccb-1c468fe5e17a.png">

**Comment by radeusgd** [comment]
This does not work if I just `Warning.attach` a warning and try to call methods on it. Something more subtle is going on here, and I don't know what exactly.

I won't be surprised if this is related to #6258.

**Comment by radeusgd** [comment]
Confirmed can still reproduce as of today.

Just small modifications to the repro code needed to update to changes:
```
from Standard.Base import all
from Standard.Table import Table, Value_Type
from Standard.Database import Database, SQLite, In_Memory

main =
    c = Database.connect (SQLite In_Memory)
    t0 = Table.new [["X", ["a", "bc", "def"]]]
    t1 = t0.select_into_database_table c "Table1"
    t2 = t1.cast "X" (Value_Type.Char size=1)
    t2.print
    warnings = Warning.get_all t2 . map .value
    IO.println warnings.length
    IO.println warnings.first
    IO.println warnings.distinct.length
```

yielding:
![image](https://github.com/enso-org/enso/assets/1436948/d9720ed2-ed10-4858-af8a-8ec123b7885e)

**Comment by JaroslavTulach** [comment]
I am getting following error when I try to execute the snippet:
```
Execution finished with an error: Method `upload_table` of type SQLite_Connection could not be found.
        at <enso> w.main(w.enso:8:10-35)
```
I am using `2023.2.1-nightly.2023.8.8`.

**Comment by radeusgd** [comment]
> I am getting following error when I try to execute the snippet:
> 
> ```
> Execution finished with an error: Method `upload_table` of type SQLite_Connection could not be found.
>         at <enso> w.main(w.enso:8:10-35)
> ```
> 
> I am using `2023.2.1-nightly.2023.8.8`.

Right, I somehow pasted the wrong (old) snippet, I don't know how I mixed them up. Updated with up-to-date snippet (`upload_table` was obsoleted and replaced by `select_into_database_table`).

**Comment by JaroslavTulach** [comment]
Yesterday I was able to reproduce the problem. Today I upgraded to 59329bd59aeacc4f60ccc4ef323dfaf6629c7710 and it seems to be working fine:

![image](https://github.com/enso-org/enso/assets/26887752/a3791e74-aa60-4ad7-bb13-e984d9c99c53)

Given there are just three fixes in between
```bash
$ git log 2023.2.1-nightly.2023.8.8...59329bd59aeacc4f60ccc4ef323dfaf6629c7710
commit 59329bd59aeacc4f60ccc4ef323dfaf6629c7710 (HEAD -> develop, origin/develop, origin/HEAD)
    New Top Bar
commit b656b336c7d239ef95a548f87deb5af2f4801368
Author: Radosław Waśko <radoslaw.wasko@enso.org>
    Report `Loss_Of_Integer_Precision` when an integer is not exactly representable as a float during conversion
commit 05e8aade1f5586f011b4943ba9fb4fa221405a56
Author: Jaroslav Tulach <jaroslav.tulach@enso.org>
Date:   Tue Aug 8 11:18:38 2023 +0200
    Exclusive context policy ensures EnsoContext.get returns compilation constant (#7493)
```
and most of #7493 actually focuses on fixing `getRootNode()` for many `@ExportedLibrary` nodes. I'd say this issue is fixed by a _collateral goodness_ of #7493.

**Comment by radeusgd** [comment]
> I'd say this issue is fixed by a _collateral goodness_ of #7493.

Awesome 🎉 and thanks for checking!
